src/app/components/pages/StudentRegistration.tsx

```diff
@@ -1,13 +1,5 @@
 import React, { useState } from "react";
-import {
-  selectors,
-  updateCurrentUser,
-  useAppDispatch,
-  useAppSelector,
-} from "../../state";
-import { Link } from "react-router-dom";
-import ReactGA from "react-ga";
-import ReactGA4 from "react-ga4";
```

Comment:
something changing in here with GA?

Reply:
I pulled the Google analytics tracking of registration into the use registration hook, since that's where the validation is done and we only want it to record successful registrations